<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>launcherRules</class>
 <widget class="QWidget" name="launcherRules">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Launcher Rules</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="connectCheckBox">
     <property name="text">
      <string>Connect tasks with this launcher</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="rulesTableWidget">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Rule</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Expression</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="detectPropertiesButton">
     <property name="text">
      <string>Detect Window Properties</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
